src/backend/gl/src/info.rs

```diff
@@ -237,28 +246,57 @@ pub fn get(gl: &gl::Gl) -> (Info, Capabilities, PrivateCaps) {
         max_texture_size: get_usize(gl, gl::MAX_TEXTURE_SIZE),
         max_patch_size: if tessellation_supported { get_usize(gl, gl::MAX_PATCH_VERTICES) } else {0},
 
-        instance_base_supported:           info.is_version_or_extension_supported(4, 2, "GL_ARB_base_instance"),
-        instance_call_supported:           info.is_version_or_extension_supported(3, 1, "GL_ARB_draw_instanced"),
-        instance_rate_supported:           info.is_version_or_extension_supported(3, 3, "GL_ARB_instanced_arrays"),
-        vertex_base_supported:             info.is_version_or_extension_supported(3, 2, "GL_ARB_draw_elements_base_vertex"),
-        srgb_color_supported:              info.is_version_or_extension_supported(3, 2, "GL_ARB_framebuffer_sRGB"),
-        constant_buffer_supported:         info.is_version_or_extension_supported(3, 1, "GL_ARB_uniform_buffer_object"),
+        instance_base_supported:           info.is_supported(Some((4, 2)),
```

Comment:
I like how structured it became, although there are holes still (e.g. with "GL_NV_copy_buffer" check).
I'd prefer this code to be less verbose though. For example, we could pass extensions as `&[&str]` list instead of `Option<&str>`. And for the GL versions we could even go for `f32` values (`3.1`, `3.2`, `NaN`).